src/object/schema_manager.c

```diff
@@ -2158,6 +2158,8 @@ sm_check_name (const char *name)
 char *
 sm_downcase_name (const char *name, char *buf, int buf_size)
 {
+#define MAX_PRINT_SIZE 64
+  int name_size = 0;
```

Comment:
Is there any need to check the length in sm_downcase_name() function? In my opinion, an existing function without a length check would suffice.

Reply:
As you said, I removed this code because the function to check the length was run before.
Thank you.